src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Commands.RecoveryServices.Backup.Helpers.csproj

```diff
@@ -11,6 +11,8 @@
     <AssemblyName>Microsoft.Azure.Commands.RecoveryServices.Backup.Helpers</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
```

Comment:
I don't think we should check-in Nuget package restore. Confirm from Anudeeb before checking in.